TodoList/Sources/TodoList.Contracts/Interfaces/IListItemUrlGenerator.cs

```diff
@@ -1,9 +0,0 @@
-﻿using TodoList.Contracts.Models;
```

Comment:
And here you lost your history. @VlastimilM, correct me if I'm wrong, but you it was your commit that doomed the file history, right? So, please, do fix this up so the whole interface lifetime can be seen. None of the commits currently available in the history of the `IListiItemUrl` in the `.Api` (70851b37d9e939e788ba8265b4a4ed847eac2f8e, 0871c8d509db380c0f4375ad087deeda2408eaf3) explains nor justifies its existence.